packages/create-llama/helpers/tools.ts

```diff
@@ -0,0 +1,25 @@
+export type Tool = {
+  display: string;
+  name: string;
+  config?: any;
+};
+
+export const supportingTools: Tool[] = [
+  {
+    display: "Google Search",
+    name: "google_search",
+    config: { engine: "", key: "", num: 2 },
```

Comment:
can we ask the config parameters if we provide the schema of the config like this?
```suggestion
    config: { engine: {question: "What is the engine?", type: "string", required: true}, key: {question: "What is the key?", type: "string", required: true} num: {type: "number", default: 2} },
```

Reply:
we do that later